x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/spatial/110_geo_shape_parameters.yml

```diff
@@ -46,6 +49,8 @@ setup:
   - match: {shapes.mappings.properties.no_doc_values_no_index.type: geo_shape }
   - match: {shapes.mappings.properties.no_doc_values_no_index.index: false }
   - match: {shapes.mappings.properties.no_doc_values_no_index.doc_values: false }
+  - match: {shapes.mappings.properties.stored.type: geo_shape }
```

Comment:
The rest of this file has tests that actually access the doc-values fields, so perhaps we could add a a test that accesses the stored field?

Reply:
I tried but the output is different depending on the XContent type. The issue is how YAML test stringify the binary data coming from a binary format like smile and  the way Json does it as [Base64](https://en.wikipedia.org/wiki/Base64). .

Reply:
OK, understood.